### PR TITLE
Cleanup group Selector

### PR DESCRIPTION
**Pull request by JordanSjodinFaithlife**
This PR contains a number of fixes:

One is it removed some duplicated code and switched from using float and hardcoded padding to center things to using flex box.

Also, if removed some styles that were being set directly on components instead of using styled components.

Lastly, making a complicated component that works on everyone's web page is hard. 'p' elements were used in this component and were getting useragent styles with margins such that it looked nice in the catalog, but could look bad elsewhere. Switching to divs works around this and seems better since the 'p' elements weren't really being used for paragraphs of text.

@dustinsoftware

### CODE REVIEW

File: components/group-selector/styled.jsx [hunk 1]
```diff
@@ -27,91 +27,67 @@ export const GroupInputButton = styled.div`
...
-	vertical-align: middle;
-	max-width: 200px;
-	width: 76%;
-`;
-
-export const SimpleGroupName = styled.div`
-	font-size: 13px;
-	color: #585250;
-	white-space: nowrap;
+	width: ${props => props.size}px;
```
Comment (dustinsoftware) [inline components/group-selector/styled.jsx]: Is the width and height being overridden anywhere? Or is this old code?

**Comment by dustinsoftware** [comment]
There are some issues with scrollbar fading in Firefox:
image

**Comment by dustinsoftware** [comment]
When mousing over Create, the button does not change state and appears to be unclickable:
image

File: components/group-selector/styled.jsx (hunk 1 above)
Comment (JordanSjodinFaithlife) [reply components/group-selector/styled.jsx]: should be set here 
https://github.com/Faithlife/styled-ui/pull/52/files/926526371bfd2bbbdc9c57ad2d9c42d620306cdb#diff-9739e43a5a0288393f4da68ba14cfefcR16

**Comment by JordanSjodinFaithlife** [comment]
The two bugs you mentioned are present in the current version. Could I address those in a future PR? (i.e. not have them block this one)

**Pull request merged**